dev/profiling.md

```diff
@@ -62,10 +70,12 @@ EXPLAIN SELECT name FROM students JOIN exams USING (sid) WHERE name LIKE 'Ma%';
                              │   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │
                              │ Filters: name>=Ma AND name│
                              │  <Mb AND name IS NOT NULL │
+                             │   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │
+                             │           EC: 1           │
                              └───────────────────────────┘
 ```
 
-We can see that the `logical_plan` contains the unoptimized query plan, involving a cross product and a `LIKE` operation. The optimized plan transforms this plan and pushes down the filters, transforming the cross product into a comparison join. It also recognizes that the `LIKE` operator only does prefix filtering, and transforms the more expensive `LIKE` operator into a cheaper `PREFIX` selection.
+Note that the query does is not actually executed – therefore, we can only see the estimated cardinality (`EC`) for each operator.
```

Comment:
Maybe add a note about what EC means / what it represents ?

Reply:
Good point, I added a (very) brief explanation.